README.md

```diff
@@ -7,61 +7,106 @@
+ - [`kubetest`][kubetest] integration also built on these packages (currently WIP).
+
+`kind` bootstraps each "node" with [kubeadm][kubeadm].
+For more details see [the design documentation][design doc].  
+
+**NOTE**: `kind` is still a work in progress.
+See [docs/todo.md] for a list of pending tasks.
+
+## Installation
+
+You can install `kind` by running:
+
+```console
+go get -u sigs.k8s.io/kind
+```
+
+## Usage
+
+To use `kind`, you need to [install Docker].
+Once you have Docker running you can create a cluster by running:
+
+```console
+kind create cluster
+```
+
+To point at the newly created cluster, you can run:
+
+```console
+export KUBECONFIG=$(kind get kubeconfig-path)
+```
+
+To delete your cluster after you're done testing, use the following command:
+
+```console
+kind delete cluster
+```
+
+## Using a custom version of Kubernetes
 
-`kind` bootstraps each "node" with [kubeadm][kubeadm]. For more details see [the design documentation][design doc].  
+To create a cluster from a specific Kubernetes version, you must ensure the following:
 
-**NOTE**: `kind` is still a work in progress, see [docs/todo.md].
+- That Kubernetes is cloned in `$(go env GOPATH)/src/k8s.io/kubernetes`.
+- That the tag/commit you want to use is checked out in the above directory.
 
-## Installation and usage
+Then, you must build the base and node images (replacing `<base-image>` and `<node-image>` as desired):
 
-You can install `kind` with `go get sigs.k8s.io/kind`
+```console
+kind build base-image --image <base-image>
+kind build node-image --base-image <base-image> --image <node-image>
+```
 
-To use `kind`, you will need to [install docker].  
-Once you have docker running you can create a cluster with `kind create cluster`  
-To delete your cluster use `kind delete cluster`
+After having performed these steps, you can launch a Kubernetes cluster by running:
 
-<!--TODO(bentheelder): improve this part of the guide-->
-To create a cluster from Kubernetes source:
-- ensure that Kubernetes is cloned in `$(go env GOPATH)/src/k8s.io/kubernetes`
-- build a node image and create a cluster with `kind build node-image && kind create cluster --image kindest/node:latest`
+```console
+kind create cluster --image <node-image>
+```
 
-For more usage see [the docs][user guide] or run `kind [command] --help`
+For more usage details check [the documentation][user guide] or run `kind [command] --help`
 
 ## Community, discussion, contribution, and support
 
-Please reach out for bugs, feature requests, and other issues!  
+Please reach out for bugs, feature requests, and other issues!
 The maintainers of this project are reachable via:
 
 - [Kubernetes Slack] in the [#kind] channel
-- [filing an issue] against this repo
+- [Filing an issue] against this repo
 - The Kubernetes [SIG-Testing Mailing List]
 
 Current maintainers (approvers) are [@BenTheElder] and [@munnerz].
 
-Pull Requests are very welcome!  
```

Comment:
this double space was intentional, I want this statement to stand out a little :-)